sale_commission_product/README.rst

```diff
@@ -0,0 +1,32 @@
+Commission By Product
```

Comment:
Can you follow current OCA's template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst?